prog/encodingexec_test.go

```diff
@@ -204,7 +210,8 @@ func TestSerializeForExec(t *testing.T) {
 			"test$array1(&(0x7f0000000000)={0x42, \"0102030405\"})",
 			[]uint64{
 				execInstrCopyin, dataOffset + 0, execArgConst, 1, 0x42,
-				execInstrCopyin, dataOffset + 1, execArgData, 5, 0x0504030201,
+				execInstrCopyin, dataOffset + 1, execArgData, 5,
+				convDataToUint64([]byte{0x01, 0x02, 0x03, 0x04, 0x05}),
```

Comment:
This is quite verbose and convoluted way to represent numbers. I will need some additional brain cycles to decode this while reading. Will it work if we add `letoh64` and then do `letoh64(0x0504030201)`?

Reply:
Hmm, i must disagree here because i find it more readable if i do not have to swap bytes every time i look at this code :) But i do not mind changing it, no big deal.

Reply:
done